mix.exs

```diff
@@ -51,6 +52,7 @@ defmodule ElixirBench.Mixfile do
       {:yamerl, "~> 0.6.0"},
       {:confex, "~> 3.3"},
       {:excoveralls, "~> 0.8", only: :test},
+      {:ex_machina, "~> 2.2", only: :test},
```

Comment:
I'd like to learn what @michalmuskala thinks here. My opinion of `ex_machina` is sort of split. The Ecto 2 book mentions we shouldn't really need those dependencies and can just build structs ourselves. I tend to agree but have never done it, as at work `ex_machina` was also introduced - I figure yearning for the known in Ruby.

Might also be of that experience and dealing with bugs in `ex_machina` that weren't fix for a long time in 1.x :)